apps/src/studio/studio.js

```diff
@@ -2804,7 +2807,7 @@ Studio.displayFeedback = function () {
       // The rails session cookie is blocked on some playlab level types,
       // causing isSignedIn to be null. In this case, use the userType (based on
       // a different cookie) to determine if the user is signed in.
-      disableSaveToGallery: level.disableSaveToGallery || !(isSignedIn || userType),
```

Comment:
why do we no longer need this `level.disableSaveToGallery` check anymore?

Reply:
this was only ever defined for one level, which was the hoc2015 free play level. Now that we're enabling save to gallery on that level, there are no remaining instances of levels with this property.